### PR TITLE
kindle-create 1.93 (new cask)

### PR DESCRIPTION
**Pull request by ghost**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [ x Checked the cask was not [already refused](https://github.com/search?q=repo%3AHomebrew%2Fhomebrew-cask+is%3Aclosed+is%3Aunmerged+&type=pullrequests) (add your cask's name to the end of the search field).
- [x] `brew audit --cask --new <cask>` worked successfully.
- [x] `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

---

### CODE REVIEW

File: Casks/k/kindle-create.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
+cask "kindle-create" do
+  version "1.93.0.0"
+  sha256 :no_check
+
+  url "https://d2bzeorukaqrvt.cloudfront.net/KindleCreateInstaller.pkg",
+      verified: "https://d2bzeorukaqrvt.cloudfront.net/"
```
Comment (khipp) [inline Casks/k/kindle-create.rb:6]:
```suggestion
      verified: "d2bzeorukaqrvt.cloudfront.net/"
```

The `verified` parameter should not have a URL scheme.

File: Casks/k/kindle-create.rb [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  name "Kindle Create"
+  desc "Creating beautiful books has never been easier"
+  homepage "https://kdp.amazon.com/en_US/help/topic/GUGQ4WDZ92F733GC"
```
Comment (khipp) [inline Casks/k/kindle-create.rb:9]:
```suggestion
  homepage "https://www.amazon.com/Kindle-Create/b?node=18292298011"
```

File: Casks/k/kindle-create.rb [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+  pkg "KindleCreateInstaller.pkg"
+
+  uninstall pkgutil:  "com.amazon.kc",
+
+  zap trash: [
+    "~/Library/Saved Application State/com.amazon.kc.savedState",
+    "/private/var/folders/rt/f0npbks941303yj_g47205f80000gn/C/com.amazon.kc",
+  ]
+
```
Comment (khipp) [inline Casks/k/kindle-create.rb:27]:
```suggestion

  caveats do
    requires_rosetta
  end
```

It does not seem to have a release targeting Apple Silicon devices.

File: Casks/k/kindle-create.rb [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+  depends_on macos: ">= :catalina"
+
+  pkg "KindleCreateInstaller.pkg"
+
+  uninstall pkgutil:  "com.amazon.kc",
+
+  zap trash: [
+    "~/Library/Saved Application State/com.amazon.kc.savedState",
+    "/private/var/folders/rt/f0npbks941303yj_g47205f80000gn/C/com.amazon.kc",
```
Comment (khipp) [inline Casks/k/kindle-create.rb:25]:
```suggestion
```

There is no need to remove the files located in `/private/var` since those are temporary files.

**Review by khipp — COMMENTED** [review]
Thank you for your submission, @jinyuzu99!

I left a few suggestions below. Please also run `brew style --fix <cask>` since there are formatting issues that can be corrected automatically.

File: Casks/k/kindle-create.rb [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+  sha256 :no_check
+
+  url "https://d2bzeorukaqrvt.cloudfront.net/KindleCreateInstaller.pkg",
+      verified: "https://d2bzeorukaqrvt.cloudfront.net/"
+  name "Kindle Create"
+  desc "Creating beautiful books has never been easier"
+  homepage "https://kdp.amazon.com/en_US/help/topic/GUGQ4WDZ92F733GC"
+
+  livecheck do
+    skip "No version information available"
```
[the remark this answers is not in the document]
Comment (jinyuzu99) [reply Casks/k/kindle-create.rb:12]: I can't find a plist of this package.

File: Casks/k/kindle-create.rb [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+
+  depends_on macos: ">= :catalina"
+
+  pkg "KindleCreateInstaller.pkg"
+
+  uninstall pkgutil:  "com.amazon.kc",
+
+  zap trash: [
+    "~/Library/Saved Application State/com.amazon.kc.savedState",
+  ]
```
Comment (khipp) [inline Casks/k/kindle-create.rb:25]:
```suggestion
  auto_updates true
  depends_on macos: ">= :catalina"

  pkg "KindleCreateInstaller.pkg"

  uninstall pkgutil: "com.amazon.kc"

  zap trash: "~/Library/Saved Application State/com.amazon.kc.savedState"
```

We still have to fix the style-related CI failures. Run `brew style --fix kindle-create.rb` to have them corrected automatically.

File: Casks/k/kindle-create.rb (hunk 5 above)
[the remark this answers is not in the document]
Comment (khipp) [reply Casks/k/kindle-create.rb:12]: The `livecheck` has to look like the following to use the `ExtractPlist` strategy:

```rb
  livecheck do
    url :url
    strategy :extract_plist
  end
```

Also, the `version` stanza has to be updated to the extracted version (`1.93` in this case).

**Review by p-linnane — APPROVED** [review]

**Review by khipp — APPROVED** [review]
Thank you, @jinyuzu99!

**Pull request merged**